src/thermo/Phase.cpp

```diff
@@ -699,7 +743,8 @@ size_t Phase::addElement(const std::string& symbol, doublereal weight,
 }
 
 bool Phase::addSpecies(shared_ptr<Species> spec) {
-    if (m_species.find(toLowerCopy(spec->name)) != m_species.end()) {
+    // species names are case sensitive
+    if (m_species.find(spec->name) != m_species.end()) {
```

Comment:
This comment seems unnecessary -- it's basically describing the contrast to the previous behavior.